types/src/v0/impls/block/full_payload/ns_proof/test.rs

```diff
@@ -0,0 +1,151 @@
+use crate::{v0::impls::block::test::ValidTest, NsProof, Payload};
+use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
+use futures::future;
+use hotshot::traits::BlockPayload;
+use hotshot_types::{
+    traits::EncodeBytes,
+    vid::{vid_scheme, VidSchemeType},
+};
+use jf_vid::{VidDisperse, VidScheme};
+
+#[async_std::test]
+async fn ns_proof() {
+    let test_cases = vec![
```

Comment:
I changed the `tests_case` variable to this:

```rust
let test_cases = vec![
        vec![vec![5, 8, 8], vec![7, 9, 11], vec![10, 5, 8], vec![7, 8, 9], vec![]],
        vec![vec![1, 2, 3], vec![4, 5, 6]],
        vec![],
    ];
```

And all tests keep passing.  So maybe we can consider this extended tests cases vector.

Reply:
Done in https://github.com/EspressoSystems/espresso-sequencer/pull/1691/commits/e52331149952c901ee14f6cba2e877ed63fc84c3

The first `vec![]` does nothing because it adds no new transactions to the vec of txs submitted to `from_transactions`. The second `vec![]` is nontrivial; it adds an empty namespace.